src/pathwayvis/services/mapoption.service.ts

```diff
@@ -164,9 +164,6 @@ export class MapOptionService {
 
   public setCurrentGrowthRate(growthRate: number) {
     this.getDataObject().mapData.map.growthRate = growthRate;
-    if (_.round(growthRate, 5) === 0) {
```

Comment:
I think this is the only instance where lodash is used in mapoption service. If so could you please delete the import?